src/parser/demonhunter/havoc/modules/features/Checklist/Component.js

```diff
@@ -43,10 +43,40 @@ class HavocDemonHunterChecklist extends React.PureComponent {
         >
           {combatant.hasTalent(SPELLS.IMMOLATION_AURA_TALENT.id) && <AbilityRequirement spell={SPELLS.IMMOLATION_AURA_TALENT.id} />}
           {combatant.hasTalent(SPELLS.FELBLADE_TALENT.id) && <AbilityRequirement spell={SPELLS.FELBLADE_TALENT.id} />}
-          {combatant.hasTalent(SPELLS.FIRST_BLOOD_TALENT.id) && <AbilityRequirement spell={SPELLS.BLADE_DANCE.id} />}
-          {combatant.hasTalent(SPELLS.FIRST_BLOOD_TALENT.id) && <AbilityRequirement spell={SPELLS.DEATH_SWEEP.id} />}
+          {combatant.hasTalent(SPELLS.FIRST_BLOOD_TALENT.id) && <AbilityRequirement spell={SPELLS.BLADE_DANCE.id} /> && <AbilityRequirement spell={SPELLS.DEATH_SWEEP.id} />}
           {combatant.hasTalent(SPELLS.MOMENTUM_TALENT.id) && <AbilityRequirement spell={SPELLS.FEL_RUSH_CAST.id} />}
           {combatant.hasTalent(SPELLS.MOMENTUM_TALENT.id) && <AbilityRequirement spell={SPELLS.VENGEFUL_RETREAT.id} />}
+        </Rule>
+
+        <Rule
+          name="Don't waste casts"
+          description={(
+            <>
+            <p>Ineffectively or improperly casting these spells will lead to dps loss</p>
```

Comment:
```suggestion
            Ineffectively or improperly casting these spells will lead to dps loss.
```
`p` tags may lead to inconsistent layout.